scripts/install.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES.
+set -euo pipefail
+
+# RAPIDS_CUDA_VERSION is like 12.15.1
+# We want cu12
+RAPIDS_PY_CUDA_SUFFIX=$(echo "cu${RAPIDS_CUDA_VERSION:-12.15.1}" | cut -d '.' -f 1)
+
+DASK_VERSION=main
+
+# Try
+uv pip install --extra-index-url=https://pypi.anaconda.org/rapidsai-wheels-nightly/simple \
+  --overrides=requirements/overrides.txt \
+  --prerelease allow \
+  "cudf-${RAPIDS_PY_CUDA_SUFFIX}" \
+  "dask-cudf-${RAPIDS_PY_CUDA_SUFFIX}" \
+  "ucx-py-${RAPIDS_PY_CUDA_SUFFIX}" \
+  "ucxx-${RAPIDS_PY_CUDA_SUFFIX}" \
+  "scipy" \
+  "dask-cuda"
+
+# Clone cudf repo for tests
+CUDF_VERSION="branch-25.04"
+cudf_commit=$(./scripts/check-version.py cudf)
```

Comment:
This bit getting `cudf_commit` and checking it out is new.

Reply:
Okay, so this takes care of the fact that the cloned version of cudf would typically be slightly different than the cudf version used in the nightly build. Seems like a good way to deal with this.